### ISSUE TITLE
.orig files cause trouble

### ISSUE THREAD

**Issue by buffstop** [yoshimkd/swift-auto-diagram#14]
If you have *.swift.orig files in a directory, I get issue #13 even parsing 4 files only.

**Comment by yoshimkd** [comment]
Thanks for the report.
I investigated what happens when files with an extension `.swfit.orig` are present in a directory. Turns out they are ignored when parsing.
I believe the issue will be the same if you remove those files from the directory. If that's what happens, you can close this issue.

**Comment by buffstop** [comment]
No. A folder with 5 .swift files and 1 ore more .swift.orig files shows the behaviour described in issue #13 (loading indicator forever). Removing the .orig files cures the issue.

**Comment by yoshimkd** [comment]
Unfortunately, I can't reproduce your issue.
This is what I'm trying:

the files in a directory:
<img width="96" alt="files-in-a-directory" src="https://user-images.githubusercontent.com/3770381/44492026-5dc58900-a663-11e8-9055-b936611e2b28.png">

the files' contents:
<img width="703" alt="file-contents" src="https://user-images.githubusercontent.com/3770381/44492042-67e78780-a663-11e8-92e6-fa7ed74f1b40.png">

the result diagram:
<img width="456" alt="result-diagram" src="https://user-images.githubusercontent.com/3770381/44492056-72a21c80-a663-11e8-855d-ef299763efaf.png">

It appears that the file with a `.swift.orig` is getting ignored as I previously stated. Let me know how I can reproduce your issue.

Also, I'm a little confused with [your previous comment](https://github.com/yoshimkd/swift-auto-diagram/issues/13#issuecomment-414758666) where you say:
```
To check if non-swift files or weird directory names cause the issue I have copied all .swift files of a mid size project to one directory (346 files). The problem persisted.
```
Judging by this statements, removing the non `.swift` files didn't fix that problem for you but in [this comment](https://github.com/yoshimkd/swift-auto-diagram/issues/14#issuecomment-415087466) a similar thing fixed that problem.

Can you please let us know in which cases the `.swift.orig` files cause problems?
